members/SGAG36eb0dbc.yaml

```diff
@@ -59,6 +59,12 @@ contact_form:
           selector: "#edit-email-mail-2"
           value: $EMAIL
           required: true
+    - javascript:
+        - name: message
+          selectors: [ "#edit-please-provide-a-brief-description-about-your-request-concern"]
+          values: [ "$MESSAGE" ]
+          commands: [ "elements[0].value = values[0].replace(/\\b\\d{5}(-\\d+})?\\b/g, '')" ]
```

Comment:
This should be `elements[0].value = elements[0].value.replace(...)` so that the value being replaced is the one that has already been truncated by the fill in step.

You can completely exclude the `values: [ "$MESSAGE" ]` as well.